test/functional/mempool_package_limits.py

```diff
@@ -177,6 +178,107 @@ def test_desc_count_limits(self):
         node.generate(1)
         assert all([res["allowed"] for res in node.testmempoolaccept(rawtxs=package_hex)])
 
+    def test_desc_count_limits_2(self):
+        """Create A bushy Package with 24 transaction in mempool and 3 transaction in package
+                              M1a
+                             ^  ^
+                            ^    M2a
+            M1b            ^       .
+              ^           ^         .
+               ^         ^           ^
+                ^       ^            M23a
+                 ^     ^
+                  ^   P1
+                   ^ ^  ^
+                    P2   P3
+        P1 has M1a as a mempool ancestor, P2 has only M1b as in-mempool ancestor whereas P3 has
+        no in-mempool ancestors, but when combined both P2 and P3 have M1a as an ancestor and
+        M1a exceeds descendant_limits (23 in-mempool descendants + 3 in-package descendants)
+        """
+
+        node = self.nodes[0]
+        package_hex = []
+        parents_tx = []
+        values = []
+        scripts = []
+        # M1a
+        first_coin_a = self.coins.pop()
+        parent_value = (first_coin_a["amount"] - Decimal("0.0002")) / 2 # Deduct reasonable fee and make 2 outputs
```

Comment:
I actually think a fee of just `Decimal("0.0001")` would be sufficient here. In fact, I think it'd be best to import `DEFAULT_FEE` from test_framework/wallet.py and use that throughout this test.